packages/expo-video/ios/VideoModule.swift

```diff
@@ -118,6 +118,45 @@ public final class VideoModule: Module {
         player.staysActiveInBackground = staysActive
       }
 
+      Property("isLooping") { (player: VideoPlayer) -> Bool in
```

Comment:
```suggestion
      Property("isLooping") { (player) -> Bool in
```
We can drop the type here